### PR TITLE
[Issue {#7}] Add new models for classification training 

### PR DESCRIPTION
**Pull request by snega16**
**This PR fixes for issue #7** 

**Changes made** 
- Added new classification model XGBoost Classifier in models.py
- Added new requirements for the model in requirements.txt

**Reason**

XGBoost Classifier model handles the missing data efficiently and it also has built in cross validation capability. It is also regularized, so the models don't overfit. To add, it also uses gradient descent algorithm to minimize loss. So this model can give good accuracy.

### CODE REVIEW

**Comment by sagnik1511** [comment]
Please add  a new line "" This PR fixes for issue #7 ""
